NewsApi/analyzer.py

```diff
@@ -0,0 +1,111 @@
+"""
+    This contains sentiment analysis models
+"""
+import json
+import pandas as pd
+import nltk
+nltk.download('vader_lexicon')
+
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+from flask import request
+from newsapi import NewsApiClient
+from NewsApi.api_key import api_key
+from sqlalchemy import null, exc
+
+
+class DataProcessor:
+    @staticmethod
+    def data_processing():
+        keyword = request.json['keyword']
+        if keyword != null and len(keyword) > 0:
+            keyword = request.json['keyword']
+            newsapi = NewsApiClient(api_key=api_key)
```

Comment:
Does this client need to be initialized on every call? I don't know how it works but I doubt that is the case